src/parse.rs

```diff
@@ -111,7 +112,7 @@ pub enum AnyNum {
 pub struct Bytes<'a> {
```

Comment:
Maybe `Bytes` isn't the best name anymore

Reply:
That’s true - what about Parser?